collector/src/execute.rs

```diff
@@ -894,6 +900,17 @@ impl<'a> Processor for BenchProcessor<'a> {
     }
 }
 
+fn store_documentation_size_into_stats(stats: &mut Stats, doc_dir: &Path) {
+    match fs_extra::dir::get_size(doc_dir) {
```

Comment:
We might also just copy-paste the `get_size` function from `fs_extra`, it's self-contained.

Reply:
I'm not sure on licensing if we copy/paste - let's try to just reimplement here. We should also have a comment describing whether we're collecting disk space size or "actual" size (i.e., does a 100 byte file take up 4KB on at least ext4)?

Reply:
I think that we should collect the actual size to have precise measurements (otherwise small changes could be lost e.g. due to rounding to 4 KiB).

I reimplemented the function and added a comment that describes the size that is gathered. I also wrote a simple test for it.